ext/linux/include/lk_cmpxchg.h

```diff
@@ -10,11 +10,11 @@
 #define __ASM_CMPXCHG_H
```

Comment:
It would be good to add some documentation here as to what was changed from the Linux kernel to be able to be used in this microbenchmark.